FpsSet.py

```diff
@@ -25,7 +25,7 @@ class FpsSet:
     relevant_fields_dict: a dictionary mapping the JSON field equivalents
     of each field to their value within the object. 
   """
-    def __init__(self, ccTLDs, primary, associated_sites=None, service_sites=None):
+    def __init__(self, ccTLDs, primary, associated_sites=[], service_sites=[]):
         self.ccTLDs = ccTLDs
         self.primary = primary
```

Comment:
Default args only override the values of arguments when no arg was passed in (e.g. `foo()` instead of `foo(x)`). So if we explicitly pass in `None` (or a variable whose value is `None`), we'll still end up storing `None` in the class members.

We also have the same issue with `ccTLDs` as for the other args, so IMO we should handle default values for that too while we're at it.

We could do something like this:

```suggestion
    def __init__(self, ccTLDs, primary, associated_sites=[], service_sites=[]):
        if ccTLDs is None:
            ccTLDs = {}
        if associated_sites is None:
            associated_sites = []
        if service_sites is None:
            service_sites = []
        self.ccTLDs = ccTLDs
        self.primary = primary
```

This way, it's much harder for callers to use an FpsSet instance in a way that's reasonable but would still break.